slacker.go

```diff
@@ -130,36 +112,6 @@ func (s *Slacker) Interactive(interactiveEventHandler func(InteractiveBotContext
 	s.interactiveEventHandler = interactiveEventHandler
 }
 
-// CustomBotContext creates a new bot context
-func (s *Slacker) CustomBotContext(botContextConstructor func(context.Context, *slack.Client, *socketmode.Client, *MessageEvent) BotContext) {
-	s.botContextConstructor = botContextConstructor
-}
```

Comment:
This breaks #91 / #103. I don't think we have enough data as to how this is used to say if we provide another means to do what they want without it.